pkgs/development/node-packages/node-packages-v10.nix

```diff
@@ -2119,22 +2119,22 @@ let
         sha512 = "tHq6qdbT9U1IRSGf14CL0pUlULksvY9OZ+5eEgl1N7t+OA3tGvNpxJCzuKQlsNgCVwbAs670L1vcVQi8j9HjnA==";
       };
```

Comment:
This commit doesn't update meguca, it's already 1.1.4 in master.

Reply:
Right of course, the previous regenerate already did that. Okay, I'll pick it out.